<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_widget</class>
 <widget class="QWidget" name="Tab_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tab_widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="pTab1">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pTab2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pTab3">
      <attribute name="title">
       <string>Tab 3</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pTab4">
      <attribute name="title">
       <string>Tab 4</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
